src/BaGet.UI/src/App.tsx

```diff
@@ -23,18 +25,49 @@ class App extends React.Component<RouteComponentProps, IAppState> {
   }
 
   public render() {
-    return (
+    return [(
       <div>
         {this._renderNavigationBar()}
 
         {this._renderContent()}
       </div>
-    );
+      ),
+      (
+        <footer className="footer">
+          {this._renderFooter()}
+        </footer>
+      )];
```

Comment:
Let's move the  `footer` element to inside the `_renderFooter` method. Also, you should be able to move the `this._renderFooter() to be next to the `_renderContent`. Something like:

```jsx
return (
  <div>
    {this._renderNavigationBar()}

    {this._renderContent()}

    {this._renderFooter()}
  </div>
);
```

Reply:
Yep, that's a good idea.